crates/ra_fmt2/src/scratch.rs

```diff
@@ -0,0 +1,42 @@
+use crate::edit_tree::EditTree;
+use crate::pattern::PatternSet;
+/// experiment purposes
+///
+// use crate::engine;
+// use crate::dsl::SpacingDsl;
+use crate::rules::spacing;
+use crate::trav_util::{walk, walk_nodes, walk_tokens};
+
+use ra_syntax::{
+    ast::{self, AstNode, AstToken},
+    Parse, SmolStr, SourceFile, SyntaxKind,
+    SyntaxKind::*,
+    SyntaxNode, SyntaxToken, T,
+};
+
+#[test]
+fn try_it() {
+    let rs_file = "pub(crate)struct Test {x: String}";
```

Comment:
Can this be formatted worse? For example use `struct\n\tTest`.

Reply:
Thanks for the advice. Is it okay if leave some unresolved as reminders to myself, I don't want to clutter up the PR or if you all have a specific workflow you want me to follow I am more than happy to accommodate any requests.
Thanks again